File: epics/css/sys-mng-opi/CSS/scripts/WriteFileExample.js
importPackage(Packages.org.csstudio.opibuilder.scriptUtil);
importPackage(Packages.java.io);

var filePath = display.getWidget("filePathWrite").getPropertyValue("text");
var text = display.getWidget("textInput").getPropertyValue("text");
FileUtil.writeTextFile(filePath, true, text, false);

File: epics/css/sys-mng-opi/CSS/scripts/WriteFileExample.py
from org.csstudio.opibuilder.scriptUtil import PVUtil
#Basic Setup

filename3 = "/home/pricardofc/CSS-Workspaces/sys-mng-opi"+display.getWidget("filePathWrite").getPropertyValue("text")
myfile = open(filename3, 'w')
myfile.write(display.getWidget("writeLabel").getPropertyValue("text"))
myfile.close()
